src/plugins/recordTypes/objectexit/optionLists.js

```diff
@@ -0,0 +1,50 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  exitReasons: {
+    values: [
+      'deaccession',
+      'disposal',
+      'returnofloan',
+      'temporary',
+      'transfer',
+      'painted out',
+      'covered by construction',
+      'lost to demolition',
+    ],
+    messages: defineMessages({
+      deaccession: {
+        id: 'option.exitReasons.deaccession',
+        defaultMessage: 'deaccession',
+      },
+      disposal: {
+        id: 'option.exitReasons.disposal',
+        defaultMessage: 'disposal',
+      },
+      returnofloan: {
+        id: 'option.exitReasons.returnofloan',
+        defaultMessage: 'return of loan',
+      },
```

Comment:
It's better not to configure the `deaccession`, `disposal`, and `returnofloan` messages here, since they'll be inherited from cspace-ui.js.

Reply:
I thought that would be the case; when testing I didn't see them but I think that's just a quirk with the devserver. I'll go ahead and get these removed.